layouts/explore-layout.vue

```diff
@@ -36,6 +36,8 @@
       <ListingCartMini />
       <ListingCartModal />
     </template>
+
+    <Loader v-model="isLoading" :can-cancel="false" />
```

Comment:
hmm.. any side effects with child component that already import this Loader?
I would like to keep logic outside `layouts`

Reply:
thinking same
could cause double loaders

Reply:
> hmm.. any side effects with child component that already import this Loader?
> I would like to keep logic outside `layouts`

I think it should be fine because they use a different `v-model` source. I moved the state and update state loader on the layout to the plugin here https://github.com/kodadot/nft-gallery/blob/feat/show-empty-collection/plugins/loader.ts



> thinking same
> could cause double loaders

As long as the child component is not set to multiple loader state, it should be fine. Better if the child component uses `$updateLoader` instead

Please note, currently I only put it on one layout. Would be better if we put a loader on all layouts

Reply:
child component should be fine I guess, can continue with this:
```vue
<template>
  <div>
    <Loader v-model="isLoading" />
    <button @click="toggleLoader()">Toggle Loader</button>
  </div>
</template>

<script lang="ts" setup>
const isLoading = ref(false)

const toggleLoader = () => {
  isLoading.value = !isLoading.value
}
</script>
```

it's better to change the child component to this:
```vue
<template>
  <div>
    <button @click="toggleLoader()">Toggle Loader</button>
  </div>
</template>

<script lang="ts" setup>
const { $loader, $updateLoader } = useNuxtApp()

const toggleLoader = () => {
  $updateLoader(!$loader.value)
}
</script>
```